base_layer/p2p/src/test_utils.rs

```diff
@@ -67,7 +67,7 @@ pub fn make_node_identity() -> Arc<NodeIdentity> {
     )
 }
 
-pub fn make_dht_header() -> DhtMessageHeader {
+pub fn make_dht_header(trace: u64) -> DhtMessageHeader {
```

Comment:
I think you should rather use the MessageTag type here to make it clear what this value is.